scripts/code-rabbit-yaml.ts

```
@@ -0,0 +1,66 @@
+    const nangoYamlInstructions = extractSection(instructionsMd, 'Configuration - nango.yaml');
+    const scriptInstructions = extractSection(instructionsMd, 'Scripts');
+    const scriptHelpers = extractSection(helpersMd, 'Script Helpers');
+
+    const additionsMap: Record<string, string> = {
+        'integrations/**/nango.yaml': nangoYamlInstructions,
+        'integrations/**/**.ts': `${scriptInstructions}\n\n# Script Helpers\n${scriptHelpers}`
+    };
+
+    const updatedYaml = appendInstructions(templateYaml, additionsMap);
+    fs.writeFileSync(OUTPUT_PATH, updatedYaml);
+    console.log(`✅ YAML generated: ${OUTPUT_PATH}`);
```

ESLint

[error] Use nango.log instead of console.log

(@nangohq/custom-integrations-linting/no-console-log)